docs/DeveloperGuide.md

```diff
@@ -703,39 +700,198 @@ testers are expected to do more *exploratory* testing.
+   
+1. Test case: `edit cs2100 c/cs2100s Computer Organisation II`<br>
+   Expected: No module edited. _Invalid module code format_ message is shown in the result display.
+   
+1. Test case: `edit cs2100 cr/4a`<br>
+   Expected: No module edited. _Invalid modular credits format_ message is shown in the result display.
+   
+1. Test case: `edit c/cs2100s cr/5`<br>
+   Expected: No module edited. _Invalid command format_ message is shown in the result display.
+      
+1. Other invalid edit commands to try: `edit`, `edit cs2103t 2103 5`, `edit 1`<br>
+   Expected: Similar to previous.
+      
+### List All Modules
 
-   1. Prerequisites: Multiple modules in the 'Current Modules'. e.g. CS2103T in 'Current Modules'.
+1. Prerequisite: 
+   1. List of 'Completed Modules' being displayed in GradPad must be filtered, e.g. using `find`.
+   1. Command must not be accompanied by any arguments.
 
-   1. Test case: `delete CS2103T`<br>
-      Expected: CS2103T module is deleted from 'Current Modules'. Details of the deleted module shown in the status message.
+1. Test case: `list`<br>
+   Expected: The full list of 'Completed Modules' is displayed.
+   
+1. Test case: `list modules`<br>
+   Expected: Full list of 'Completed Modules' is not displayed, current list is unchanged. _Invalid command format_ message is shown in the result display.
 
-   1. Test case: `delete AA1000`<br>
-      Expected: No module is deleted. Error details shown in the status message.
+### Find a Specific Module or a Group of Modules
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` <br>
-      Expected: Similar to previous.
+1. Prerequisites: 
+   1. Arguments must be specified.
+   1. Module to be included must exist in the list being displayed in GradPad, e.g. CS2100, CS2101, CS3230 and ST2334 are in the list and CS2106 is not.
+      
+1. Test case: `find cs2`<br>
+   Expected: CS2100 and CS2101 are displayed.
+   
+1. Test case: `find cs2 st`<br>
+   Expected: CS2100, CS2101 and ST2334 are displayed.
+   
+1. Test case: `find cs3230`<br>
+   Expected: CS3230 is displayed.
+
+1. Test case: `delete cs2106`<br>
+   Expected: No modules displayed.
+      
+1. Test case: `find`<br>
+   Expected: Current list is unchanged. _Invalid command format_ message is shown in the result display.
+   
+### Check Total Modular Credits
+
+1. Prerequisite: Command must not be accompanied by any arguments.
+
+1. Test case: `checkmc`
+   Expected: Total modular credits are calculated and displayed. If there are no modules in 'Completed Modules', total modular credits will be 0.
+   
+1. Test case: `checkmc modules`
+   Expected: Total modular credits are not calculated. _Invalid command format_ message is shown in the result display.
+   
+### Open Help Page
+
+1. Prerequisite: Command must not be accompanied by any arguments.
+
+1. Test case: `help`
+   Expected: Help page is displayed.
+   
+1. Test case: `checkmc modules`
```

Comment:
Is there a typo for `help modules`? LOL actually can't tell cos this is still a correct invalid command HAHA

Reply:
oh ya its supposed to be help modules LOL cause its under "### Open Help Page". i'll fix it!